src/main/kotlin/uk/gov/justice/probation/courtcaseservice/service/listeners/notifiers/OffenderCreatedEventNotifier.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.probation.courtcaseservice.service.listeners.notifiers
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.probation.courtcaseservice.client.OffenderDetailRestClient
+import uk.gov.justice.probation.courtcaseservice.client.model.listeners.DomainEvent
+import uk.gov.justice.probation.courtcaseservice.service.ProbationCaseEngagementService
+
+const val NEW_OFFENDER_CREATED = "probation-case.engagement.created"
+
+@Component(value = NEW_OFFENDER_CREATED)
+class OffenderCreatedEventNotifier(
```

Comment:
This isn't doing a notifier job, as in not notifying anything.. processing (processor) an event instead?